### ISSUE TITLE
EM304 mistracking up and down slopes

### ISSUE THREAD

**Issue by shoy-NOAA** [oceanmapping/community#25]
There is an ongoing issue with significant consistent nadir mistracking going up and down slopes (especially on seamounts). We reached out to Kongsberg but have been unable to alleviate this issue. We will troubleshoot further during our upcoming shakedown.

**Comment by ejheffron** [comment]
On EX2303 we noticed extensive misstracking up and down slopes. In this case we did have the Penetration Filter on and set to Medium. We have turned it off and so far no major misstracking. Kongsberg recommends having this OFF unless you specifically observe mistracking. The attached image is what misstracking would look like in the Water Column display.
![ExampleMistracking_fromKM](https://github.com/oceanmapping/community/assets/80911286/36ded5c3-3e07-431c-a4c9-dc94478bc620)
